### DISCUSSION TITLE
Modifier regex

### DISCUSSION THREAD

**Discussion by benjaminpreiss** [tailwindlabs/tailwindcss#12528] [Help]
Hey Y'all

I was wondering if someone can point me in the direction where I can find the official regex for tailwind modifiers (if there is such a thing).

Meaning: A regex that detects only the modifierspart given a string.

e.g. `hover:md:` in ` hover:md:bg-blue-500` or `min-[320px]:hover:` in `min-[320px]:hover:bg-blue-500`

**Comment by benjaminpreiss** [comment]
If anyone is wondering - this is the regex not for the modifiers, but for the detection of any tailwind class: https://github.com/tailwindlabs/tailwindcss/blob/b52a78b56f329af8714bf2664a673bd2f2b2e336/src/lib/defaultExtractor.js#L3C17-L3C33

**Reply by wongjn** [reply]
I believe this bit may be the modifiers regex: https://github.com/tailwindlabs/tailwindcss/blob/8350cffdad34efb6625e0f2b0ff738edaf0d1cce/src/lib/defaultExtractor.js#L92-L113
